ampersand-form-view.js

```diff
@@ -50,13 +56,9 @@ module.exports = View.extend({
         // add all our fields
         (result(opts, 'fields') || result(this, 'fields') || []).forEach(this.addField, this);
 
-        if (opts.autoRender) {
```

Comment:
This was a bit confusing -- the README says that `autoRender` defaults to `true`, but this code suggests otherwise. I'll post an issue about that. For now I just have it doing what it was before, except there's never a need to set `this.template` to `true` because there's now an actual `template` property on the class.